api/src/main/java/org/openmrs/validator/ConditionValidator.java

```diff
@@ -0,0 +1,28 @@
+package org.openmrs.validator;
+
+import org.openmrs.Condition;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+import org.springframework.validation.Validator;
+
+public class ConditionValidator implements Validator {
+
```

Comment:
Add comments that describe the purpose of your class.

Reply:
@CeciliaCaroline I have done those. Coming in my next push